targets/hasura/migrations/1613499638772_alert_clean/up.sql

```diff
@@ -6,7 +6,7 @@ END;
 $body$
 LANGUAGE 'plpgsql';
 
-COMMENT ON FUNCTION delete_old_alerts(duration) IS $body$
+COMMENT ON FUNCTION delete_old_alerts() IS $body$
 Remove alerts that are older that a given duration
```

Comment:
that are older than 3 months du coup ?

Reply:
la fonction accede un nombre de mois  en parametre, les 3 mois c'est dans l'utilisation du trigger plus bas